### ISSUE TITLE
Code Table Request - REMOVE bill length

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6318]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

bill length claims to be:

```
A measure of a bird's bill other than that described in <a href="https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#culmen_length" target="_blank">culmen length</a>.
```

I don't understand how this is necessary, why culmen length + method isn't better for this task.

Can we better document or merge?

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

[temp_billlen.csv.zip](https://github.com/ArctosDB/arctos/files/11537817/temp_billlen.csv.zip)

Users:

 guid_prefix | count 
-------------+-------
 ALMNH:Bird  |     1
 ASNHC:Bird  |     4
 CHAS:Bird   |   145
 CHAS:Teach  |    16
 CRCM:Bird   |     1
 DMNS:Bird   |     1
 UCM:Bird    |   216
 UMNH:Bird   |    50
 UMZM:Bird   |   106
 UTEP:Bird   |     3
 UWYMV:Bird  |     4
 UWZM:Bird   |  1002

 @ebraker
 @jessicatir
 @ewommack
 @kderieg322079
 @mvzhuang
 @makaylaeasley
 @wellerjes
 @adhornsby
 @babogan
 @droberts49
 @acdoll

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by ewommack** [comment]
I think we actually have to keep it. You are hitting up against the fact there are several different ways to measure the bill in birds, and they are all accepted in some form.
From Pyle, 1997:

> " As with the wing, at least three methods of measuring the bill length currently are employed, depending on the point at the base of the bill from which the measurement is taken...." 

He goes on to describe a measurement called: 
bill from nares to tip
exposed culmen
and
total culmen

And that's just the NAmerican standard guide for passerine banding. There are undoubtably other standards for other groups of species and other scientific research. 
When a historic tag simply says "bill length", we're not going to be able to squeeze it into culmen length and be accurate or honest.

And it won't do to squeeze it into culmen and add method either. The culmen is only part of the bill, the bill includes other structures.

**Comment by jessicatir** [comment]
I’d echo what @ewommack said; I think it should be kept.

**Comment by Jegelewicz** [comment]
> several different ways to measure the bill in birds,

That is METHOD and belongs there. @dustymc is making a point here that length of the bill is an attribute - HOW it was measured belongs in the attribute method. If we do things that way, then anyone can find ALL of the bill lengths in one place, then sort them by method.

**Comment by ewommack** [comment]
It is isn't method. The culmen is only one part of the bill. You cannot have an attribute that says culmen and then a method that measures the entire bill.

You can have a culmen length have multiple different ways to measure, defined by which banding station or researcher you are working under. 

You can also have bill length and multiple different ways to measure it. 

If you want to use method that is fine, but you still need more attributes. They are measuring different parts of the birds body.

**Comment by adhornsby** [comment]
I prefer having culmen as a separate attribute -- it's a commonly used and well-defined feature, and I wouldn't want to increase the chances of it getting confused with various other bill length measurements.

If we update the bill length definition, can we direct people to include a method? E.g., "Length of the bill (specify method). Not to be confused with culmen_length"

For whatever it's worth, I know for sure that 98% of the UMZM bill length measurements are actually culmen, and it's on my to-do list to update those records.
